components/automate-ui/src/app/modules/token/token-list/api-token-list.component.ts

```diff
@@ -87,6 +87,7 @@ export class ApiTokenListComponent implements OnInit, OnDestroy {
       .subscribe(state => {
         this.creatingToken = false;
         if (state === EntityStatus.loadingSuccess) {
+          this.assignPolicies();
```

Comment:
Previously, this call (inline) to dispatch AddPolicyMembers was done at the same time the CreateToken action was dispatched. By moving it here, we now wait for a successful notification of token creation before starting the policy additions.